client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java

```diff
@@ -149,6 +150,8 @@ public class NarayanaLRAClient implements Closeable {
     private static final long LEAVE_TIMEOUT = Long.getLong("lra.internal.client.leave.timeout", CLIENT_TIMEOUT);
     private static final long QUERY_TIMEOUT = Long.getLong("lra.internal.client.query.timeout", CLIENT_TIMEOUT);
 
+    private static final Config CONFIG = ConfigProvider.getConfig();
```

Comment:
Remark: I think caching this looks safe due to https://github.com/microprofile/microprofile-config/blob/3.0.2/spec/src/main/asciidoc/configprovider.asciidoc?plain=1#L40

Reply:
This is what I relied our tests would catch.